.github/workflows/build-tags.yml

```diff
@@ -66,14 +66,10 @@ env:
   IMAGE_REPO_DSPO: data-science-pipelines-operator
   IMAGE_REPO_SERVER: ds-pipelines-api-server
   IMAGE_REPO_UI: ds-pipelines-frontend
-  IMAGE_REPO_CACHE: ds-pipelines-cacheserver
   IMAGE_REPO_PA: ds-pipelines-persistenceagent
   IMAGE_REPO_SWF: ds-pipelines-scheduledworkflow
-  IMAGE_REPO_VC: ds-pipelines-viewercontroller
-  IMAGE_REPO_ARTIFACT: ds-pipelines-artifact-manager
```

Comment:
same as below

Reply:
https://github.com/opendatahub-io/data-science-pipelines-operator/pull/590#discussion_r1515120404

Reply:
we did not use these